hugegraph-dist/docker/scripts/try-storage.groovy

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.hugegraph.HugeFactory
+import org.apache.hugegraph.dist.RegisterUtil
+
+// register all the backend to avoid changes if docker needs to support othre backend
+RegisterUtil.registerPlugins()
+RegisterUtil.registerRocksDB()
+RegisterUtil.registerCassandra()
+RegisterUtil.registerScyllaDB()
+RegisterUtil.registerHBase()
+RegisterUtil.registerMysql()
+RegisterUtil.registerPalo()
+RegisterUtil.registerPostgresql()
+
+graph = HugeFactory.open('./conf/graphs/hugegraph.properties')
```

Comment:
can we rename this file to `detect-storage.groovy`?

Reply:
Sure.  And "detect" seems better than "try". I have applied the change.